src/ensembl/src/content/app/species-selector/components/species-selector-app-bar/SpeciesSelectorAppBar.tsx

```diff
@@ -78,7 +78,9 @@ const SelectedSpeciesList = (props: Props) => {
     />
   ));
 
-  return <SpeciesTabsWrapper speciesTabs={selectedSpecies} />;
+  return (
+    <SpeciesTabsWrapper isWrappable={false} speciesTabs={selectedSpecies} />
+  );
```

Comment:
Whoa! The idea was that both in the species selector and on the species page, the selected species lozenges are allowed to wrap over multiple lines in order to show the full names of the selected species. In other apps, such as genome browser or entity viewer, species lozenges can only ever occupy a single line, and thus can shrink and crop off parts of species names.

It should be up to the rest of the Species Selector and Species page to adapt to the growing height of the selected species bar.

Reply:
@azangru Thanks for pointing this out. I had misunderstood this task. I've pushed the changes after fixing this properly and also have vertically aligned the app bar aside.